<ui version="4.0" >
 <class>DMXAddressTool</class>
 <widget class="QDialog" name="DMXAddressTool" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>311</width>
    <height>194</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>DMX Address Tool</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="m_dipGroup" >
     <property name="title" >
      <string>Binary DIP address</string>
     </property>
     <layout class="QGridLayout" >
      <item row="0" column="0" >
       <widget class="QCheckBox" name="m_256" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QCheckBox" name="m_128" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="2" >
       <widget class="QCheckBox" name="m_64" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QCheckBox" name="m_32" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QCheckBox" name="m_16" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QCheckBox" name="m_8" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <widget class="QCheckBox" name="m_4" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="7" >
       <widget class="QCheckBox" name="m_2" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="8" >
       <widget class="QCheckBox" name="m_1" >
        <property name="text" >
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>256</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLabel" name="label_2" >
        <property name="text" >
         <string>128</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="label_3" >
        <property name="text" >
         <string>064</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" >
       <widget class="QLabel" name="label_4" >
        <property name="text" >
         <string>032</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4" >
       <widget class="QLabel" name="label_5" >
        <property name="text" >
         <string>016</string>
        </property>
       </widget>
      </item>
      <item row="1" column="5" >
       <widget class="QLabel" name="label_6" >
        <property name="text" >
         <string>008</string>
        </property>
       </widget>
      </item>
      <item row="1" column="6" >
       <widget class="QLabel" name="label_7" >
        <property name="text" >
         <string>004</string>
        </property>
       </widget>
      </item>
      <item row="1" column="7" >
       <widget class="QLabel" name="label_8" >
        <property name="text" >
         <string>002</string>
        </property>
       </widget>
      </item>
      <item row="1" column="8" >
       <widget class="QLabel" name="label_9" >
        <property name="text" >
         <string>001</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="m_decimalGroup" >
     <property name="title" >
      <string>Decimal address</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QSpinBox" name="m_decimalSpin" >
        <property name="minimum" >
         <number>1</number>
        </property>
        <property name="maximum" >
         <number>511</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="m_buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::NoButton|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DMXAddressTool</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DMXAddressTool</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
